src/xcode/ENA/ENA/Source/Scenes/HTMLViewController/HTMLViewController.xib

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Cant read this code. Now I need to open Interface Builder. Already afraid, that it will take 1 bazillion days to load. 🥺🤪 Don't worry, I just don't like Interface Builder anymore.